tutorials/getting-started/popup.js

```diff
@@ -0,0 +1,24 @@
+// Initialize butotn with users's prefered color
+let changeColor = document.getElementById("changeColor");
+
+chrome.storage.sync.get("color", ({ color }) => {
+  changeColor.style.backgroundColor = color;
+});
+
+// When the button is clicked, inject setPageBackgroundColor into current page
+changeColor.addEventListener("click", async () => {
+  let [tab] = await chrome.tabs.query({ active: true, currentWindow: true });
```

Comment:
Event handlers aren't really async but this is probably fine. It might be worth makin a note after the comment on line 8.

```js
// Event handlers are not async, but querying tabs is, so errors will be raised via unhandledrejection
```